setup.py

```diff
@@ -22,6 +23,15 @@
     'xxhash',
 ]
 
+# This will determine which compiler is being used to build the C modules
+compiler_type = distutils.ccompiler.get_default_compiler()
+if compiler_type in ("unix", "mingw32"):
+    compiler_optimize_flags = ['-O3', '-ftree-vectorize']
+elif compiler_type == "msvc":
+    compiler_optimize_flags = ['/Ox', '/Ob2', '/Oi', '/Ot', '/Oy', '/GL']
```

Comment:
```suggestion
    compiler_optimize_flags = ['/Ox', '/GL']
```

Note that according to [here](https://learn.microsoft.com/en-us/cpp/build/reference/ox-full-optimization?view=msvc-170#remarks), `/Ox` is the same as using `/Ob2`, `/Oi`, `/Ot`, and `/Oy` in combination, so I believe we can just remove these.